docs/architecture/ADR-005-QGB_posting_confirms_in_reserved_namespace.md

```diff
@@ -0,0 +1,96 @@
+# ADR 5: Using block space instead of state to post attestations confirms
+
+## Changelog
+
+- {date}: {changelog}
+
+## Context
+
+Currently, the QGB is using the state solely to save all the attestations and confirms it needs.
```

Comment:
[nit]
```suggestion
Currently, the QGB is using the state to save all the attestations and confirms it needs.
```

Rationale: Even if confirms are removed from state, there are other things stored in state due to the attestations:

> Then, the only state that the QGB module will keep is the one that is created in [EndBlocker](https://github.com/celestiaorg/celestia-app/blob/a965914b8a467f0384b17d9a8a0bb1ac62f384db/x/qgb/abci.go#L12-L16).
Which are `Attestations`, i.e. `Valset`s and `DataCommitmentRequest`s.